client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -38,79 +38,17 @@ const CalendarPage = () => {
     setModal(!showModal);
   };
   const [currDate, setCurrDate] = useState(new Date());
-  const children = [
-    {
-      title: "GDSC Meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: false,
-    },
-    {
-      title: "another meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: true,
-    },
-    {
-      title: "GDSC Meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: false,
-    },
-    {
-      title: "another meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: true,
-    },
-    {
-      title: "GDSC Meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: false,
-    },
-    {
-      title: "another meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: true,
-    },
-    {
-      title: "GDSC Meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: false,
-    },
-    {
-      title: "another meeting",
-      locationText: "DH2020",
-      startTimeText: "05:00PM",
-      endTimeText: "07:00PM",
-      isOngoing: true,
-    },
-  ];
-  const cards = children.map((task, ix) => (
-    <li key={ix} style={{ listStyle: "none" }}>
-      <TaskCard
-        title={task.title}
-        location={task.locationText}
-        startTime={task.startTimeText}
-        endTime={task.endTimeText}
-        isOngoing={task.isOngoing}
-        showDetailsDialog={() => {
-          console.log("show the modal from here");
-          toggleModal();
-        }}
-      />
-    </li>
-  ));
+  const sampleTask = {
+    title: "gdsc meeting",
+    location: "dh2020",
+    startTime: "05:00pm",
+    endTime: "07:00pm",
+    isOngoing: false,
+  };
+  const sampleTasks = [];
+  for (let i = 0; i < 15; i++) {
+    sampleTasks.push({ ...sampleTask });
```

Comment:
In this particular example, we won't need to actually use the spread operator. Generally we would only use it if we need to make another copy of an existing state and making any modifications. Since this is only a placeholder for the sampleTasks, this can stay for now!